cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -139,6 +139,17 @@ export_list_to_bazel(
 # Handle the dependencies between features. That is, if feature "X" is enabled
 # also enable feature "Y" because "X" depends on "Y".
 function (google_cloud_cpp_enable_deps)
+    if (__ga_libraries__ IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
+        list(REMOVE_ITEM GOOGLE_CLOUD_CPP_ENABLE __ga_libraries__)
+        list(APPEND GOOGLE_CLOUD_CPP_ENABLE ${GOOGLE_CLOUD_CPP_GA_LIBRARIES})
+        list(APPEND GOOGLE_CLOUD_CPP_ENABLE
+             ${GOOGLE_CLOUD_CPP_TRANSITION_LIBRARIES})
+    endif ()
+    if (__experimental_libraries__ IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
```

Comment:
Are `__ga_libraries__` and `__experimental_libraries__` public? (I am assuming yes, and that the documentation is coming later).

I claim that no one would ever build `GOOGLE_CLOUD_CPP_ENABLE=__experimental_libraries__`.

So I would prefer offering `GOOGLE_CLOUD_CPP_ENABLE=__experimental_and_ga_libraries__` as a way to build all libraries. (I wish we could say `__all_libraries__`, but we must force the customers to type the dreaded letters: "experimental")

Reply:
Yes, they are public.  I am thinking about where the documentation will go.

Reply:
Fixed the documentation too.  And yes, I want folks to have to type the dreaded letters.

Reply:
I see you opted not to document `__experimental_libraries__`. I guess that one is an easter egg.